wger/nutrition/templates/ingredient/form.html

```diff
@@ -1,6 +1,6 @@
 {% load i18n %}
 
-{% if not perms.exercises %}
```

Comment:
This should actually have been `perms.nutrition.add_ingredient`

Reply:
Gotcha, since we changed the permissions of a "normal" user we were updating all the places that were dependent on the old exercise check and this was one of them, didn't realize it wasn't supposed to be perms.exercises in the first place!